src/sinks/gcp/pubsub.rs

```diff
@@ -49,14 +49,22 @@ impl SinkBatchSettings for PubsubDefaultBatchSettings {
 #[derive(Clone, Debug)]
 pub struct PubsubConfig {
     /// The project name to which to publish events.
+    #[configurable(metadata(docs::examples = "vector-123456"))]
     pub project: String,
 
     /// The topic within the project to which to publish events.
+    #[configurable(metadata(docs::examples = "this-is-a-topic"))]
     pub topic: String,
 
     /// The endpoint to which to publish events.
-    #[serde(default)]
-    pub endpoint: Option<String>,
+    /// The scheme (`http` or `https`) must be specified. No path should be included since the paths defined
+    /// by the [`GCP Pub/Sub`][pubsub_api] api are used.
+    /// The trailing slash `/` must not be included.
```

Comment:
```suggestion
    /// 
    /// The scheme (`http` or `https`) must be specified. No path should be included since the paths defined
    /// by the [`GCP Pub/Sub`][pubsub_api] api are used.
    ///
    /// The trailing slash `/` must not be included.
```